auto-rename-tag.el

```diff
@@ -47,6 +47,9 @@
 (defvar-local auto-rename-tag--pre-command-actived nil
   "Check if `pre-command-hook' called.")
 
+(defvar-local auto-rename-tag--after-action-active nil
```

Comment:
The other variable is called `actived`, I'm not sure if the `d` is meant for something? Perhaps you meant `activated`? 

I can rename this one or the other if you like.

Reply:
Oh, yes. That's the typo! Good catch! Thanks!